profile/README.md

```diff
@@ -0,0 +1,32 @@
+_API Integrations the right way_
+
+[![Discord](https://img.shields.io/discord/819563244418105354?logo=discord&logoColor=fff)](https://sfc.is/discord)
+[![NPM](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2Fsuperfaceai)](https://twitter.com/superfaceai)
```

Comment:
This image is confusing

- it states "Tweet" while leading me to the Twitter profile
- alt for it states "NPM"
- the style doesn't match with the Discord one